src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java

```diff
@@ -250,21 +257,18 @@ private void printProcessHeader(Map<String, HazelcastProcess> processes) {
         }
     }
 
-    private void printProcessEntry(
-            @Option(names = {"-n", "--names"}, description = "Shows names only") boolean namesOnly,
-            @Option(names = {"-r", "--running"}, description = "Shows running members only") boolean runningOnly,
-            HazelcastProcess process) {
+    private void printProcessEntry(boolean namesOnly, boolean runningOnly, HazelcastProcess process) {
         int pid = process.getPid();
         String processName = process.getName();
         if (namesOnly) {
             if (!runningOnly || process.getStatus() == RUNNING) {
                 println(processName);
             }
-        } else if (process.getStatus() != STOPPED) {
+        } else if (runningOnly && process.getStatus() == RUNNING) {
             printf(LIST_FORMAT,
                     processName, pid, process.getStatus(), process.getCreationInstant(), process.getClusterName());
```

Comment:
The format for CREATED is not really human readable. Can we format this?
`2019-09-23T11:31:51.121Z`

Reply:
👍